src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -112,10 +121,6 @@ class Quantizer(Compressor):
     Base quantizer for pytorch quantizer
     """
 
-    def __call__(self, model):
```

Comment:
why removing this func?

Reply:
There is an exactly same version already in parent class Compressor.

Reply:
i see